host/ecallids.h

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#ifndef _OE_HOST_ECALLIDS_H
+#define _OE_HOST_ECALLIDS_H
+
+#include <openenclave/edger8r/host.h>
+#include <openenclave/host.h>
+
+#define OE_ECALL_ID_NULL OE_UINT64_MAX
+#define OE_MAX_ECALLS 256
```

Comment:
Why do you need a fixed maximum instead of being able to realloc dynamically?  256 seems like it runs the danger of a regression if there are complicated EDLs out there.

Reply:
Good point. I have removed the maximum limit, and used realloc instead.